### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_About_Zoom: At reflow resolution "About" link is not visible in the page.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspnetcore#49703]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopWeb; #BM_.NETCore_Web_Jul23; #.NET Core; #WCAG1.4.10; #Win11; #Reflow; #ChromiumEdge; #A11yWCAG2.1;

### Environment Details:
App name: .NET Core
URL: http://localhost:5268/

Chromium Edge: Version 116.0.1938.29 (Official build) dev (64-bit)

### Repro Steps:
Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
Move to the created directory using command "cd foldername".
Run command "dotnet new blazorwasm" and hit ENTER key.
Run command "dotnet run".
Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.

TAB to "About" link and observe that "About" link is not visible in the page.

### Actual Result:
At reflow resolution "About" link is not visible in the page.

### Expected Result:
At reflow resolution "About" link should be visible clearly in the page.

### User Impact:
Low vision users will not be able to see the link and will not get information about.

**Comment by merriemcgaw** [comment]
@mkArtakMSFT @SteveSandersonMS - this is required for RC2, can someone please take a look?

**Comment by mkArtakMSFT** [comment]
#Closed;

**Comment by merriemcgaw** [comment]
@PoojaNamde this is the last of the bugs opened during the test pass. Please validate.

**Comment by PoojaNamde** [comment]
#Reopened; #Regressed:09-15-23;

Verified this issue in the environment http://localhost:5227, but issue is still able to repro. hence reactivating the bug.

![Screenshot (720)](https://github.com/dotnet/aspnetcore/assets/90901519/aeb67889-29b6-4e53-9a78-482ba736a4e8)

**Comment by merriemcgaw** [comment]
Please retest on Tuesday with the latest build of .NET 8. It will be on [this page](https://github.com/dotnet/installer) in the middle column.

**Comment by PoojaNamde** [comment]
We will verify once .NET RC2 build will be available.

**Comment by kupatkar99** [comment]
#Closed; #Regressed:10-04-23;
Verified the issue in latest environment
Browser details: Microsoft Edge
Version 119.0.2132.0 (Official build) dev (64-bit)
Issue is fixed. Hence Closing the bug. Please find added attachment for reference.
![49703_Fixed](https://github.com/dotnet/aspnetcore/assets/98530069/ef7105ed-ddd3-451a-8ec8-0584232bd043)
